src/Driver/DriverInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Equip\Queue\Driver;
+
+interface DriverInterface
```

Comment:
These are the minimum methods required for a driver, but I'm certain I will need to add more.